website/pages/docs/starknet/starknet-artifacts.mdx

```diff
@@ -0,0 +1,38 @@
+# Starknet Artifacts
+
+As part of building Starknet contracts, Scarb generates a `<package_name>.starknet_artifacts.json` file
+containing a machine-readable data about built artifacts.
+
+Version 1 of this file has a structure like this:
+
+```json
+{
+  "version": 1,
+  "contracts": [
+    {
+      "id": "randomid1",
+      "package_name": "mypackage",
+      "contract_name": "Contract2",
+      "artifacts": {
+        "sierra": "mypackage_Contract2.sierra.json",
+        "casm": null
+      }
+    },
+    {
+      "id": "randomid2",
+      "package_name": "mypackage",
+      "contract_name": "Contract1",
+      "artifacts": {
+        "sierra": "mypackage_Contract1.sierra.json",
+        "casm": null
+      }
+    }
+  ]
+}
+```
+
+- `package_name` is the name of the package defined in `Scarb.toml`
```

Comment:
Please tell users that they should use `id` to refer to items in this list. It has the highest chance of being unique